packages/fast-components-react-base/src/image/image.tsx

```diff
@@ -67,38 +66,24 @@ class Image extends Foundation<IImageHandledProps & IManagedClasses<IImageClassN
             return (
                 <picture
                     {...this.unhandledProps()}
-                    className={this.generateClasses()}
+                    className={super.generateClassNames(get(this.props, "managedClasses.image__picture"))}
```

Comment:
this should still recieve the root class - `managedClasses.image`. A modifier should never exist without a block